### PR TITLE
Added RMoT Invitationals page.

### PR DESCRIPTION
**Pull request by oralekin**
Added the page for RMoT Invitationals#2, only in English.

### CODE REVIEW

File: wiki/Tournaments/RMoT_Invitational_#2/en.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# RMoT Invitationals 2
+
+The Rapid Monthly osu! Tournament Invitationals #2 is a tournament organized and ran by the Rapid Tournaments team.
+It is an exclusive tournament where only players who reached semi-finals in one of the last 4 RMoT's get invited.
+
+## Schedule
```
Comment (kj415j45) [inline wiki/Tournaments/RMoT_Invitational_#2/en.md]: markdown table crashed here

Comment (oralekin) [inline wiki/Tournaments/RMoT_Invitational_#2/en.md]: It was fine on the preview, GitHub broke it :/

Comment (sr229) [inline wiki/Tournaments/RMoT_Invitational_#2/en.md]: what previewer do you use? @oralekin

Comment (oralekin) [inline wiki/Tournaments/RMoT_Invitational_#2/en.md]: VS Code's native previewer

Comment (RaikireHiuduo) [inline wiki/Tournaments/RMoT_Invitational_#2/en.md]: Separate it with a newline whitespace between the two.

So:-
```Markdown
The RMoT Invitationals are traditionally held over two weekends.

| Event                               | Date        |
|------------------------------------:|:------------|
| Round of 16                         | 6 May 2017  |
| Quarterfinals and Losers Round 1    | 7 May 2017  |
| Semifinals and Losers Round 2 and 3 | 13 May 2017 |
| Losers Round 4, 5 and 6             | 14 May 2017 |
| Finals and Grand Finals             | 14 May 2017 |
```

File: wiki/Tournaments/RMoT_Invitational_#2/en.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+| Mappool Selectors | [![](/wiki/shared/flag/Th.gif)GuChangeDeWorld](), [![](/wiki/shared/flag/Ca.gif)Meow-Mix]()                                                                                                                                                                         |
+| Streamers         | [![](/wiki/shared/flag/Se.gif)BeatingHeart](), [![](/wiki/shared/flag/Pl.gif)FreezingTooEasy]()                                                                                                                                                                     |
+| Commentators      | [![](/wiki/shared/flag/Pl.gif)bsurma](), [![](/wiki/shared/flag/Se.gif)ErroTheCube](), [![](/wiki/shared/flag/Ph.gif)kristoffer09](), [![](/wiki/shared/flag/De.gif)Niegth](), [![](/wiki/shared/flag/Si.gif)ultraregen]()                                          |
+
+## Prizes
+
+The Prizes are handed out in the form of Points. 36 points can be redeemed for a month of osu! Supporter. Invitational winners are awarded twice the amount awarded in RMoT's.
+The winner of the Invitational tournament gets 96 points, 2<sup>nd</sup> place gets 48 points etc..
```
Comment (smoogipooo) [inline wiki/Tournaments/RMoT_Invitational_#2/en.md]: We are not using points or awarding supporter anymore.

Comment (oralekin) [inline wiki/Tournaments/RMoT_Invitational_#2/en.md]: @smoogipooo we have our own "point" system for redeeming supporter, we can change the name to Rapid points or something if it's a problem.

**Comment by MegaApplePi** [comment]
From some testing with osu-web, it seems that using hashtags (`#`) in the folder name will cause it to do unexpected things (see below). **Does the folder name really need to use the hashtag (`#`)?**

----

Since hashtags is a reserved keyword, it has to be escaped using percent encoding. So we must type:
```
https://new.ppy.sh/wiki/Tournaments/RMoT_Invitational_%232/
                                                      ^^^
```
instead of:
```
https://new.ppy.sh/wiki/Tournaments/RMoT_Invitational_#2/
                                                      ^
```

Otherwise, osu-web will parse it as
```
https://new.ppy.sh/wiki/Tournaments/RMoT_Invitational_/#2
                                                     ^^^
```
and will try to link to id `2` in page `RMoT_Invitational_` which will 404 anyways.

File: wiki/Tournaments/RMoT_Invitational_#2/en.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+| Mappool Selectors | [![](/wiki/shared/flag/Th.gif)GuChangeDeWorld](), [![](/wiki/shared/flag/Ca.gif)Meow-Mix]()                                                                                                                                                                         |
+| Streamers         | [![](/wiki/shared/flag/Se.gif)BeatingHeart](), [![](/wiki/shared/flag/Pl.gif)FreezingTooEasy]()                                                                                                                                                                     |
+| Commentators      | [![](/wiki/shared/flag/Pl.gif)bsurma](), [![](/wiki/shared/flag/Se.gif)ErroTheCube](), [![](/wiki/shared/flag/Ph.gif)kristoffer09](), [![](/wiki/shared/flag/De.gif)Niegth](), [![](/wiki/shared/flag/Si.gif)ultraregen]()                                          |
+
+## Prizes
+
+The Prizes are handed out in the form of Points. 36 points can be redeemed for a month of osu! Supporter. Invitational winners are awarded twice the amount awarded in RMoT's.
+The winner of the Invitational tournament gets 96 points, 2ⁿᵈ place gets 48 points etc..
```
Comment (kj415j45) [inline wiki/Tournaments/RMoT_Invitational_#2/en.md]: `2nd` may ok, don't need to up `nd`

File: wiki/Tournaments/RMoT_Invitational_#2/en.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+| Mappool Selectors | [![](/wiki/shared/flag/Th.gif)GuChangeDeWorld](), [![](/wiki/shared/flag/Ca.gif)Meow-Mix]()                                                                                                                                                                         |
+| Streamers         | [![](/wiki/shared/flag/Se.gif)BeatingHeart](), [![](/wiki/shared/flag/Pl.gif)FreezingTooEasy]()                                                                                                                                                                     |
+| Commentators      | [![](/wiki/shared/flag/Pl.gif)bsurma](), [![](/wiki/shared/flag/Se.gif)ErroTheCube](), [![](/wiki/shared/flag/Ph.gif)kristoffer09](), [![](/wiki/shared/flag/De.gif)Niegth](), [![](/wiki/shared/flag/Si.gif)ultraregen]()                                          |
+
+## Prizes
+
+The Prizes are handed out in the form of Points. 36 points can be redeemed for a month of osu! Supporter. Invitational winners are awarded twice the amount awarded in RMoT's.
```
Comment (kj415j45) [inline wiki/Tournaments/RMoT_Invitational_#2/en.md]: osu!supporter

File: wiki/Tournaments/RMoT_Invitational_#2/en.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # RMoT Invitationals 2
 
-The Rapid Monthly osu! Tournament Invitationals #2 is a tournament organized and ran by the Rapid Tournaments team.
+The Rapid Monthly osu! Tournament Invitationals /#2 is a tournament organized and ran by the Rapid Tournaments team.
```
Comment (kj415j45) [inline wiki/Tournaments/RMoT_Invitational_#2/en.md]: `\` not `/`
:p

File: wiki/Tournaments/RMoT_Invitational_#2/en.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+| Mappool Selectors | [![](/wiki/shared/flag/Th.gif)GuChangeDeWorld](), [![](/wiki/shared/flag/Ca.gif)Meow-Mix]()                                                                                                                                                                         |
+| Streamers         | [![](/wiki/shared/flag/Se.gif)BeatingHeart](), [![](/wiki/shared/flag/Pl.gif)FreezingTooEasy]()                                                                                                                                                                     |
+| Commentators      | [![](/wiki/shared/flag/Pl.gif)bsurma](), [![](/wiki/shared/flag/Se.gif)ErroTheCube](), [![](/wiki/shared/flag/Ph.gif)kristoffer09](), [![](/wiki/shared/flag/De.gif)Niegth](), [![](/wiki/shared/flag/Si.gif)ultraregen]()                                          |
+
+## Prizes
+
+The Prizes are handed out in the form of Points. every 36 points can be redeemed for a month of osu!supporter. Invitational winners are awarded twice the amount awarded in RMoT's.
+The winner of the Invitational tournament gets 96 points, 2ⁿᵈ place gets 48 points etc..
```
Comment (kj415j45) [inline wiki/Tournaments/RMoT_Invitational_#2/en.md]: use `2nd`
don't use `2ⁿᵈ`

File: wiki/Tournaments/RMoT_Invitational_2/en.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
+# RMoT Invitationals 2
+
+The Rapid Monthly osu! Tournament Invitationals \#2 is a tournament organized and ran by the Rapid Tournaments team.
+It is an exclusive tournament where only players who reached semi-finals in one of the last 4 RMoT's get invited.
+
+The tournament brackets can be found [on Challonge.](https://challonge.com/rmotinvitational2)
+[![](/wiki/shared/flag/Pl.gif)MagBlood](https://osu.ppy.sh/u/6178640) won the Invitational by beating [![](/wiki/shared/flag/Pl.gif)bsurma](https://osu.ppy.sh/u/509575) in the finals and
```
Comment (RaikireHiuduo) [inline wiki/Tournaments/RMoT_Invitational_2/en.md]: and...?

File: wiki/Tournaments/RMoT_Invitational_2/en.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
+The Prizes are handed out in the form of Points. every 36 points can be redeemed for a month of osu!supporter. Invitational winners are awarded twice the amount awarded in RMoT's.
+The winner of the Invitational tournament gets 96 points, 2ⁿᵈ place gets 48 points etc..
+
+## Players
+
+16 players are admitted into each Invitational. Players eligible for Invitational 2 are:
+
+* [![](/wiki/shared/flag/Us.gif)_Zyrenn](https://osu.ppy.sh/u/8389508)
+* [![](/wiki/shared/flag/Sp.gif)TheMightyNeko](https://osu.ppy.sh/u/7469737)
```
Comment (RaikireHiuduo) [inline wiki/Tournaments/RMoT_Invitational_2/en.md]: `* [TheMightyNeko](https://osu.ppy.sh/u/7469737)`

File: wiki/Tournaments/RMoT_Invitational_2/en.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+* [![](/wiki/shared/flag/Es.gif)EIVO](https://osu.ppy.sh/u/8111395)
+* [![](/wiki/shared/flag/Ar.gif)Linkel](https://osu.ppy.sh/u/4267947)
+* [![](/wiki/shared/flag/Tr.gif)Koizumi Hanayo](https://osu.ppy.sh/u/1988584)
+* [![](/wiki/shared/flag/Pl.gif)Missownik](https://osu.ppy.sh/u/7714761)
+* [![](/wiki/shared/flag/No.gif)AlosChan](https://osu.ppy.sh/u/8240328)
+* [![](/wiki/shared/flag/Es.gif)-ZackFair](https://osu.ppy.sh/u/4388820)
+* [![](/wiki/shared/flag/Ca.gif)~~Chanigan~~](https://osu.ppy.sh/u/3266120)
+* [![](/wiki/shared/flag/Id.gif)akirakai18](https://osu.ppy.sh/u/2956814)
+
+AkaiKage and Chanigan gave up their places in the Invitationals and EIVO and Koizumi Hanayo reached semi-finals twice. Due to this, 4 people who have not reached semi-finals were selected to participate in the tournament, filler players.
```
Comment (MegaApplePi) [inline wiki/Tournaments/RMoT_Invitational_2/en.md]: can you start with:
```
**Notice:**
```
I didn't see this :/

**Comment by smoogipooo** [comment]
I've hijacked this PR a little bit to add RMoT to the main tournaments page, since the post was given approval I'm going to go ahead and merge this.

**Pull request merged**